### DISCUSSION TITLE
input value of Backlash Compensation

### DISCUSSION THREAD

**Discussion by compoundMR** [grblHAL/core#338] [Q&A] [answered]
i've been activated backlash compensation , but where should i input the value of backlash compensation on X,Y or Z axis ? is it in $160,161,162 or what ?

**Comment by terjeio** [comment, marked as answer]
It is $160,161,162 ...

If on Windows you may use [ioSender or the config app](https://github.com/terjeio/ioSender/releases/tag/2.0.43) for a more advanced settings interface than most other senders provide.

**Reply by compoundMR** [reply]
yea im using  UGS and  actually the $161,162.163 wont be displayed unless backlash compensation on config.h is activated
